.travis.yml

```diff
@@ -26,6 +26,7 @@ before_install:
   - "test $TRAVIS_NODE_VERSION != '0.6' || npm rm --save-dev istanbul"
   - "test $TRAVIS_NODE_VERSION != '0.8' || npm rm --save-dev istanbul"
   - "test $(echo $TRAVIS_NODE_VERSION | cut -d. -f1) -ge 4 || npm rm --save-dev $(grep -E '\"eslint\\S*\"' package.json | cut -d'\"' -f2)"
+  - "test -z $(echo $HTTP2_TEST) || npm install --only=dev https://github.com/visionmedia/superagent.git"
```

Comment:
Please only install deps from npm.

Reply:
Unfortunately, superagent with http2 have not released. I will wait for release.

Reply:
Also what does the only flag do? According to https://docs.npmjs.com/cli/install it only does something if you do not provide a module name.

Reply:
This is my mistake. `--only=dev` flag does nothing here. I should delete this flag.